webapp/components/post_view/post_list.jsx

```diff
@@ -171,14 +175,20 @@ export default class PostList extends React.PureComponent {
     }
 
     componentDidUpdate(prevProps) {
+        const prevPosts = prevProps.posts;
```

Comment:
It's definitely checking for when certain props/state change, for example `this.previousScrollHeight !== postList.scrollHeight && posts[0].id === prevPosts[0].id`. The post list doesn't re-render very often and really is a few if statements going to slow it down? The current implementation on master already does a lot of this